packages/serverless-orchestration/src/ServerlessSpoke.js

```diff
@@ -86,70 +85,31 @@ spoke.post("/", async (req, res) => {
       message: "Process exited with error 🚨",
       childProcessIdentifier: _getChildProcessIdentifier(req),
       jsonBody: req.body,
-      execResponse: execResponse instanceof Error ? execResponse.message : execResponse,
+      error: execResponse instanceof Error ? execResponse.message : execResponse,
     });
     await delay(waitForLoggerDelay); // Wait a few seconds to be sure the the winston logs are processed upstream.
     res.status(500).send({
       message: "Process exited with error",
       childProcessIdentifier: _getChildProcessIdentifier(req),
-      execResponse: execResponse instanceof Error ? execResponse.message : execResponse,
+      error: execResponse instanceof Error ? execResponse.message : execResponse,
     });
   }
 });
 
-function _execShellCommand(cmd, inputEnv, strategyRunnerSpoke = false) {
+function _execShellCommand(cmd, inputEnv) {
   return new Promise((resolve) => {
     const { stdout, stderr } = exec(
       cmd,
       { env: { ...process.env, ...inputEnv }, stdio: "inherit", maxBuffer },
```

Comment:
I'm additionally curious about the fact we set `stdio: "inherit"` here whilst _also_ reading `stdout` and `stderr` from `exec`. I think the use of inherit is probably spamming the console on the serverless instance. This is useful when debugging, but idk whether we actually want that in prod. Should we consider toggling this based on some env var?

Reply:
I'm not sure it does anything. Please double check me, but I think stdio is not actually a valid option for the exec function. It's only valid for spawn and fork.

Unless you disagree, I say we remove, but I don't think that will actually fix anything :)

Reply:
Ah, you're totally right. +1 for removing it.